### ISSUE TITLE
path traversal of "cargo contract new"

### ISSUE THREAD

**Issue by taskooh** [paritytech/cargo-contract#34]
I think "cargo contract new" should only allow alphabets , dash('-') and  numbers.

when I run this in Desktop,
```bash
Desktop % cargo contract new ../xxx
	Created contract ../xxx
```

directory "xxx" is created in home directory.
```bash
Desktop % cd ../xxx
xxx %
```

when I try to build from here,

```bash
cargo contract build
 [1/4] Collecting crate metadata
error: Error during execution of `cargo metadata`: error: failed to parse manifest at `/Users/xf9x/xxx/Cargo.toml`

Caused by:
  Invalid character `.` in package name: `../xxx`
```

build fails.

**Comment by ascjones** [comment]
This should be straighforward: this check just needs to be enhanced: https://github.com/paritytech/cargo-contract/blob/master/src/cmd/new.rs#L27

**Comment by ascjones** [comment]
Closed in #192
